pallet/xrpl/src/mock.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright 2022-2023 Futureverse Corporation Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// You may obtain a copy of the License at the root of this project source code
+
+use crate::{self as pallet_xrpl, *};
+use frame_support::{parameter_types, weights::WeightToFee, PalletId};
+use frame_system::EnsureRoot;
+use seed_pallet_common::test_prelude::*;
+use seed_primitives::{AccountId, Address, AssetId, Balance, Signature};
+use sp_core::H256;
+use sp_runtime::{generic, testing::Header, traits::LookupError};
```

Comment:
A lot of these imports are included within the test_prelude
```suggestion
use frame_support::weights::WeightToFee;
use seed_pallet_common::test_prelude::*;
use seed_primitives::{Address, Signature};
use sp_runtime::{generic, traits::LookupError};
```

Reply:
addressed: https://github.com/futureversecom/trn-seed/pull/758/commits/35a472cc9b964c12d07c40db75f56892b851bd3b